web/src/buddy_list.ts

```diff
@@ -203,52 +207,56 @@ export class BuddyList extends BuddyListConf {
+                            const tooltip_text = $t(
                                 {
                                     defaultMessage:
                                         "{N, plural, one {# participant} other {# participants}}",
                                 },
                                 {N: participant_count},
                             );
+                            instance.setContent(tooltip_text);
                         } else if (elem_id === "buddy-list-users-matching-view-section-heading") {
-                            const users_matching_view_count =
-                                non_participant_users_matching_view_count();
-                            if (narrow_state.stream_sub()) {
-                                tooltip_text = $t(
-                                    {
-                                        defaultMessage:
-                                            "{N, plural, one {# other subscriber} other {# other subscribers}}",
-                                    },
-                                    {N: users_matching_view_count},
-                                );
-                            } else {
-                                tooltip_text = $t(
-                                    {
-                                        defaultMessage:
-                                            "{N, plural, one {# participant} other {# participants}}",
-                                    },
-                                    {N: users_matching_view_count},
-                                );
-                            }
+                            void (async () => {
+                                let tooltip_text;
+                                const users_matching_view_count =
+                                    await non_participant_users_matching_view_count();
+                                if (narrow_state.stream_sub()) {
+                                    tooltip_text = $t(
+                                        {
+                                            defaultMessage:
+                                                "{N, plural, one {# other subscriber} other {# other subscribers}}",
+                                        },
+                                        {N: users_matching_view_count},
+                                    );
+                                } else {
+                                    tooltip_text = $t(
+                                        {
+                                            defaultMessage:
+                                                "{N, plural, one {# participant} other {# participants}}",
+                                        },
+                                        {N: users_matching_view_count},
+                                    );
+                                }
+                                instance.setContent(tooltip_text);
+                            })();
```

Comment:
Product-wise, how does tippy handle a promise here -- can the `instance` end up garbage-collected while we're waiting for the server to reply?

Reply:
Relatedly, I did confirm that if I set network to "on mars", we end up showing a blank tooltip, which looks broken. I think we want to either:
- Show a temporary "Fetching subscribers..."  string that we'll replace once we have the answer.
- Just have the tooltip say "25 total subscribers" if we don't yet have the subscriber data, using the `total_human_subscribers_count` we definitely have.

(I know we usually, but this is the state users will see if they've lost network)

Reply:
I went with "n total subscribers", putting the change in its own commit in case that feels better (and for ease of review) but I can also squash it if you want.

I also confirmed that setting the tooltip text after a timeout doesn't raise an exception (or open a tooltip).